### DISCUSSION TITLE
Can't make more than 253 clients

### DISCUSSION THREAD

**Discussion by Iman-Sh** [wg-easy/wg-easy#616] [Q&A] [answered]
Hi, i need to be able create more clients than 253 which is currently max allowed clients. i've tried adding:
-e WG_DEFAULT_ADDRESS=10.8.x.x to the run command but even though i get no error in running the command panel won't come up using serverIP:51821.
any idea whether is possible to increase maximum users?

**Comment by mitrios-hub** [comment, marked as answer]
> Hi, i need to be able create more clients than 253 which is currently max allowed clients. i've tried adding: -e WG_DEFAULT_ADDRESS=10.8.x.x to the run command but even though i get no error in running the command panel won't come up using serverIP:51821. any idea whether is possible to increase maximum users?

look here: https://github.com/wg-easy/wg-easy/issues/430

the WG-EASY scripts doesn't support CIDR/16 but wireguard does

**Reply by pheiduck** [reply]
If you really want to consider this, you are welcome to create a PR.
